.rubocop.yml

```diff
@@ -4,6 +4,10 @@ AllCops:
     - 'tmp/**/*'
     - 'vendor/**/*'
 
+# FIXME: Re-enable once private_attr_reader is no longer used
+Lint/UselessAccessModifier:
```

Comment:
Yes, it is an easy fix. My reasoning was that we will be touching the same locations again in #804, adding back the `private` keyword that would be removed now.